### PR TITLE
Add module option in run_tests

### PR DESCRIPTION
**Pull request by feihujiang**
Signed-off-by: feihujiang <jiangfeihu@huawei.com>

Usage:
  ./bin/run_tests [-m Module]
Options:
  -h          print usage and exit
  -m Module   only run tests of module, <core, validator, mktplace, bond, arcade>

### CODE REVIEW

**Comment by feihujiang** [comment]
@vaporos Could you please review this PR?  Running ./bin/run_tests totally costs too much time, this PR could run tests of specified module.

File: bin/run_tests [hunk 1]
```diff
@@ -7,56 +31,122 @@ top_dir=$(cd $(dirname $(dirname $0)) && pwd)
...
+        MODULE=$OPTARG
+        ;;
+      \?)
+        echo "Invalid option: -$OPTARG" >&2
+        exit 2
+        ;;
+    esac
+done
+
+case $MODULE in
```
Comment (vaporos) [inline bin/run_tests]: Let's use the same indent here as above (4 spaces instead of 2).

File: bin/run_tests [hunk 2]
```diff
@@ -7,56 +31,122 @@ top_dir=$(cd $(dirname $(dirname $0)) && pwd)
...
+  all)
+    All
```
Comment (vaporos) [inline bin/run_tests]: Let's rename Core, Validator, etc. functions to 'lint_core', 'lint_validator', etc.  Let's also remove 'All' and just call each of these "lint_" functions directly from the case statement.

File: bin/run_tests (hunk 1 above)
Comment (feihujiang) [inline bin/run_tests]: Done.

File: bin/run_tests (hunk 2 above)
Comment (feihujiang) [inline bin/run_tests]: I renamed  Core, Validator, etc. functions to ```test_core```, ```test_validator```, and removed ```All```. Done.

**Comment by feihujiang** [comment]
@vaporos I udpated this PR, please take a look.

**Pull request merged**